src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import classNames from 'classnames';
+import { TodoFilter } from '../TodoFilter';
+
+interface Props {
+  filterBy: string;
```

Comment:
Consider using a more specific type for the filter instead of string, e.g., an enumeration to restrict the filter values.